resources/terraform/aws-eks/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "region" {
+  type        = string
+  description = "The region in which the cluster is hosted."
+}
+
+variable "kubernetes_version" {
+  type        = string
+  description = "The Kubernetes version."
+  default     = "1.22"
+}
+
+
+variable "kubeconfig_path" {
+  type        = string
+  description = "Path to the kubeconfig file."
+  default     = "./kubeconfig"
+}
```

Comment:
I don't think we want this variable here since we always want to generate the `kubeconfig` inside the module root. I would remove this and use `path.module` to ensure it gets created where it should be. See https://github.com/dominodatalab/testenv/blob/3275dc58c941410a021285b0acba681cd8350ef4/resources/terraform/gcp-gke/main.tf#L71-L75 for the details.

This is also means you'll need to change the output. See https://github.com/dominodatalab/testenv/blob/3275dc58c941410a021285b0acba681cd8350ef4/resources/terraform/gcp-gke/outputs.tf#L1-L5